src/state/members-candidates/reducers.js

```diff
@@ -43,15 +43,15 @@ const peopleReducer = (state = initialState, {
         ...state,
         selectedMember: {
           ...state.selectedMember,
-          moc_links: set(state.selectedMember.moc_links ? state.selectedMember.moc_links : {}, `${payload.id}`, { id: payload.id, ...payload.link })
+          helpful_links: set(state.selectedMember.helpful_links ? state.selectedMember.helpful_links : {}, `${payload.id}`, { id: payload.id, ...payload.link })
         }
       }
     case 'EDIT_LINK':
       return {
         ...state,
         selectedMember: {
           ...state.selectedMember,
-          moc_links: mapValues(state.selectedMember.moc_links, (link) => {
+          helpful_links: mapValues(state.selectedMember.helpful_links, (link) => {
```

Comment:
i would still like you to stop using snake case in javascript.

Reply:
when you save this in the DB they can be snake_case, but in the reducer, use camelCase